src/connecpy/wsgi.py

```diff
@@ -173,38 +174,111 @@ def read_chunked(input_stream):
     return body
 
 
-class ConnecpyWSGIApp(base.ConnecpyBaseApp):
+class ConnecpyWSGIApplication:
     """WSGI application for Connecpy."""
 
-    def __init__(self, interceptors=None):
+    def __init__(self, *, path: str, endpoints: Mapping[str, base.Endpoint]):
```

Comment:
We don't have sync interceptors so for now I left them out, it's easy to add them when we properly support it.